index.html

```diff
@@ -21,21 +21,31 @@
   <!-- The header section for mobile -->
   <header>
     <a href="#" class="logo">Grace Abike</a>
-    <nav class="mobile">
-      <img src="./imgs/Menumobile.png" alt="menu">
-    </nav>
+    <div id="hamburger-icon" class="mobile" onclick="toggleMenu(this)">
+      <div id="bar1" class="bar"></div>
+      <div id="bar2" class="bar"></div>
+      <div id="bar3" class="bar"></div>
+      <div id="mobileNav" class="overlay mobile mobile-menu">
+        <div class="overlay-content">
+          <a class="bar-link1" href="#main-section">Portfolio <span class="heavy-right">&#x276F;</span></a>
+          <a class="bar-link2" href="#work">About <span class="heavy-right">&#x276F;</span></a>
```

Comment:
- [ ] It would be better if the navigation links takes us to the right paths. However, now when I click on the portfolio link it takes me to the #main-section as shown in the screenshot below

![Screenshot from 2022-09-05 17-49-36](https://user-images.githubusercontent.com/46388113/188498418-4b6b13b0-3b82-440a-abf9-c4417792bbd4.png)

Also the about link takes me to the #work path as seen in the screenshot below

![Screenshot from 2022-09-05 17-51-03](https://user-images.githubusercontent.com/46388113/188498514-2dfef05f-b8e4-46af-866d-76adbb1bbc60.png)


Let us kindly fix it since this was addressed in the previous review

Reply:
@addod19thanks for your comments I really appreciate,
but where do you want Portfolio to take you to